### PR TITLE
Route example for docs

### PR DESCRIPTION
**Pull request by betsy-lichtenberg**

### CODE REVIEW

File: examples/routes/variables.tf [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "project_id" {
+  description = "The ID of the project where the routes will be created"
+}
+
+variable "network_name" {
```
Comment (morgante) [inline examples/routes/variables.tf:21]: Please don't include variables you're not using. In this case, the only variable you referenced is "project_id."

**Review by morgante — CHANGES_REQUESTED** [review]

File: examples/routes/versions.tf [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  required_version = ">= 0.13.0"
+  required_providers {
+    google = {
+      source  = "hashicorp/google"
+      version = "<4.0,>= 2.12"
+    }
+  }
+
+  provider_meta "google" {
+    module_name = "blueprints/terraform/terraform-google-network:routes/v3.2.0"
```
Comment (morgante) [inline examples/routes/versions.tf:27]: This is not necessary.

File: examples/routes/variables.tf (hunk 1 above)
Comment (betsy-lichtenberg) [reply examples/routes/variables.tf]: Fixed

File: examples/routes/versions.tf (hunk 2 above)
Comment (betsy-lichtenberg) [reply examples/routes/versions.tf]: Fixed

**Review by morgante — APPROVED** [review]

**Pull request merged**